enterprise/app/filter/filter.tsx

```diff
@@ -169,6 +197,22 @@ export default class FilterComponent extends React.Component<FilterProps, State>
     });
   }
 
+  private onSortByToggle(sortBy: string, selected: string) {
+    selected = sortBy;
+    router.setQuery({
+      ...Object.fromEntries(this.props.search.entries()),
+      [SORT_BY_NAME]: selected,
+    });
+  }
+
+  private onSortOrderToggle(sortOrder: string, selected: string) {
```

Comment:
nit: replace `Toggle` with `Change` in these 2 function names since it's changing the value of a radio selection rather than toggling a single checkbox element

Reply:
Done.